### DISCUSSION TITLE
Should I post the issue or PR link to any channels in discord after opening it?

### DISCUSSION THREAD

**Discussion by AnushaDeviR** [EddieHubCommunity/support#8227] [First-Timers] [answered]
Hello hello, 

Hope you all are well. This is my very first step towards open-source and would like to clarify a tiny doubt (not sure if this has been discussed somewhere or not, if so please let me know). After I have opened a new issue or PR on the EddiHub repos, is there a platform on discord where I have to share to let the maintainers or community know I have raised it?

Thanks, 
Anusha

**Comment by Vijaykv5** [comment, marked as answer]
Hi Anusha

When participating in open-source projects in**EddiHub** or anyother repos, there's no need to constantly inform maintainers or the community about every issue or pull request (PR) you open. GitHub, the platform commonly used for open-source development, has built-in notification systems. When you create an issue or PR, GitHub automatically notifies the project's maintainers.

However, if you wish to draw attention to your contribution or seek quicker feedback, consider leaving a comment on your PR or issue. In this comment, you can politely ask for reviews or feedback, or you can specifically mention maintainers using their GitHub usernames with the "@" symbol.
You can always join EddiHub has a [Discord serve](https://discord.com/invite/jZQs6Wu)r for more updates.

**Reply by AnushaDeviR** [reply]
Hi Vijay, 

Thank you for your helpful reply, this clears my doubts.
